include/beast/core/detail/is_call_possible.hpp

```diff
@@ -13,41 +13,6 @@
 namespace beast {
```

Comment:
This file now appears empty, save for the `#include <type_traits>`. Given that it's a `detail`, should it just be entirely removed?